modules/services/random-background.nix

```diff
@@ -6,6 +6,12 @@ let
 
   cfg = config.services.random-background;
 
+  flags = lib.concatStringsSep " " [
+    (lib.optionalString cfg.noXinerama "--no-xinerama")
```

Comment:
Better use `optional` to avoid the unnecessary empty string. Something like
```
flags = concatStringsSep " " (
  [ "--randomize" "--bg-${cfg.display}" ]
  ++ optional (!cfg.enableXinerama) "--no-xinerama"
);
```